SabreTools.FileTypes/Enums.cs

```diff
@@ -1298,6 +1300,10 @@ public enum OutputFormat
         TorrentXZ,
         TorrentXZRomba,
         TapeArchive,
+        
+        CHD,
```

Comment:
If you're really wanting to add this, don't add CHD to this list. Adding CHD to this list means that we'd be _building_ a CHD on output.

Reply:
yeah thats what it does...

Reply:
That's not something that I feel should be in the scope of SabreTools. External tools like `chdman` make more sense to handle that.